src/pages/event-date-schema.ts

```diff
@@ -0,0 +1,14 @@
+import Joi from "joi";
+
+const maxDate = new Date(2099, 11);
+const minDate = new Date(2000, 0);
+
+export const eventDateSchema = Joi.date()
```

Comment:
I have moved this schema out of stage-page validation file and up one level as it is now shared with the new page